plugins/scaffolder/src/legacy/TemplateEditorPage/CustomFieldExplorer.tsx

```diff
@@ -154,11 +150,13 @@ export const CustomFieldExplorer = ({
           <CardContent>
             <Form
               showErrorList={false}
+              // @ts-ignore
```

Comment:
this looks like it is hiding an actual bug.  There is no variable or argument for `fieldOverrides`

Reply:
Yep interesting - nice catch